tools/istio-docker.mk

```diff
@@ -80,6 +80,7 @@ docker.eurekamirror: $(ISTIO_DOCKER)/pilot-test-eurekamirror
 docker.pilot:        $(ISTIO_DOCKER)/pilot-discovery
 docker.proxy docker.proxy_debug: $(ISTIO_DOCKER)/pilot-agent
 $(foreach FILE,$(PROXY_JSON_FILES),$(eval docker.proxy docker.proxy_debug: $(ISTIO_DOCKER)/$(notdir $(FILE))))
+docker.proxy docker.proxy_debug: $(ISTIO_DOCKER)/envoy_bootstrap_tmpl.json
```

Comment:
The $(ISTIO_DOCKER)/envoy_bootstrap_tmpl.json could be added to the end of the existing "docker.proxy docker.proxy_debug:" that's just two lines up.

Reply:
Same - I'm trying to have more readable makefile, and I'm planning to refactor the dockerfile creation (moving few files around) so it's more obvious what is going on.